apps/rush/src/actions/InstallAction.ts

```diff
@@ -127,6 +127,7 @@ export default class InstallAction extends CommandLineAction {
     stopwatch.stop();
     console.log(colors.green(`Done. (${stopwatch.toString()})`));
 
+    this._collectTelemetry(stopwatch, true);
```

Comment:
should we also collect it for failures? (so we could get a count, for example)

---
In reply to: [117844599](https://github.com/Microsoft/web-build-tools/pull/207#discussion_r117844599) [](ancestors = 117844599)